R/PlatinumGenomes-QC/Sample-Level-QC.Rmd

```diff
@@ -377,12 +424,13 @@ To entirely remove a genome from a variant set in the Genomics API:
 
 # Summary
 
-Let's wrap up with a quick comparison using all the variables we've collected for each sample:
+Let's wrap up with a quick comparison using a few the variables we've collected for each sample.
```

Comment:
There's a typo there: "a few the", but I would recommend just making this "using the variables".